src/pages/vendor/Menu.tsx

```
@@ -51,6 +58,7 @@ interface CustomOption {
 }
 
 interface MenuItem {
+  internalName: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)